src/screencast/wlr_screencast.c

```diff
@@ -545,28 +545,90 @@ static void wlr_remove_output(struct xdpw_wlr_output *out) {
 	free(out);
 }
 
+static void wlr_format_modifier_pair_add(struct xdpw_screencast_context *ctx,
+		uint32_t format, uint64_t modifier) {
+	struct xdpw_format_modifier_pair *fm_pair = calloc(1, sizeof(struct xdpw_format_modifier_pair));
+
+	fm_pair->fourcc = format;
+	fm_pair->modifier = modifier;
+
+	logprint(TRACE, "wlroots: format %u (%lu)", fm_pair->fourcc, fm_pair->modifier);
+
+	wl_list_insert(&ctx->format_modifier_pairs, &fm_pair->link);
```

Comment:
It might be easier to use a `wl_array` for the format+modifier list, instead of using a `wl_list`.

Reply:
`wl_list` currently works. I would put this onto the to refactor list

Reply:
Addressed in #209